<ui version="4.0" >
 <class>MetakonCfgForm</class>
 <widget class="QWidget" name="MetakonCfgForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Последовательный порт</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="leSerialPortName" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Скорость  передачи</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QComboBox" name="cbBaudRate" >
     <property name="editable" >
      <bool>false</bool>
     </property>
     <property name="currentIndex" >
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Адрес устройства</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QSpinBox" name="sbDevAddress" >
     <property name="maximum" >
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Управляющий канал</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QSpinBox" name="sbChannel" >
     <property name="maximum" >
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Постоянная интегр.</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QSpinBox" name="sbIntegr" >
     <property name="suffix" >
      <string>сек</string>
     </property>
     <property name="prefix" >
      <string/>
     </property>
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <number>30000</number>
     </property>
     <property name="singleStep" >
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>Постоянная диффер.</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="QSpinBox" name="sbDiff" >
     <property name="suffix" >
      <string>сек</string>
     </property>
     <property name="maximum" >
      <number>255</number>
     </property>
     <property name="singleStep" >
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="label_7" >
     <property name="text" >
      <string>Зона пропорциональности</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" >
    <widget class="QSpinBox" name="sbPb" >
     <property name="suffix" >
      <string>°C</string>
     </property>
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <number>9999</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
